.travis.yml

```diff
@@ -7,29 +7,23 @@ git:
 matrix:
   include:
     - os: linux
-      dist: xenial
+      dist: ubuntu
       sudo: required
       compiler: gcc
-      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON -DEXIV2_ENABLE_CURL=ON -DCMAKE_CXX_STANDARD=98"
+      env: COVERAGE=0 CMAKE_OPTIONS="-CMAKE_CXX_STANDARD=98 -DCMAKE_BUILD_TYPE=Debug -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=OFF -DBUILD_WITH_COVERAGE=OFF -DEXIV2_ENABLE_CURL=ON -DCMAKE_CXX_FLAGS=-Wno-deprecated"
```

Comment:
Hi Robin. The problem now with this is a missing 'D' in `-CMAKE_CXX_STANDARD=98`. 

I think today I will have some time in the evening, so that I can take a look to the existing failures at CI to help you with this.